src/Akka.Hosting.API.Tests/verify/CoreApiSpec.ApproveCluster.verified.txt

```diff
@@ -65,6 +76,7 @@ namespace Akka.Cluster.Hosting
     {
         public ClusterOptions() { }
         public string? AppVersion { get; set; }
+        public bool ClusterReadyCheck { get; set; }
```

Comment:
Added a config-based option for automatically enabling the `WithAkkaClusterReadinessCheck`